shed/index.go

```diff
@@ -41,6 +41,9 @@ type Item struct {
 	AccessTimestamp int64
 	StoreTimestamp  int64
 	BinID           uint64
+	TreeSize        uint64 // Used to store the merkle tree size in the pinIndex
+	PinCounter      uint64 // maintains the no of time a chunk is pinned
+	IsRaw           uint8  // defines if the root hash stored is from a RAW upload or is it a Manifest
```

Comment:
I'm not sure that we should store such high-level information on indexes

Reply:
Well with this logic BinID is also high level information.. 

Pin counter is absolutely needed to store the number of guys pinned a chunk..
TreeSize is required to show in list command how much a content a pinning node has pinned
IsRAW - as explained above.. i don't see any other good way to reduce complexity of pinning a file vs manifest.

Reply:
These are moved to state store now..  but pinCounter exists and is required. If you are not satisfied with this.. please let me know how to tackle this.